wordstats/cognate_db.py

```diff
@@ -0,0 +1,107 @@
+from sys import stdout
+
+import sqlalchemy.orm
+import sqlalchemy
+from sqlalchemy import Column, Integer, String, UniqueConstraint, Boolean
+from sqlalchemy import Float
+
+from .base_service import Base, SimplifiedQuery
+
+
+class UnknownCognateInfo(object):
+    """ Default values if  we have no information about
+    a given word """
+    def __init__(self):
+        self.blacklist = False
+        self.whitelist = False
+
+
+class CognateCandidatesInfo(SimplifiedQuery, Base):
+    __tablename__ = 'candidates_info'
+    __table_args__ = {'mysql_collate': 'utf8_bin'}
+
+    id = Column(Integer, primary_key=True)
+    method = Column(String(255), nullable=False, index=True)
+
+    word_from = Column(String(255), nullable =False, index = True)
+    word_to = Column(String(255), nullable=False, index=True)
+    language_ids = Column(String(4), nullable =False, index = True)
```

Comment:
not all languages have two-letter codes: https://github.com/zeeguu-ecosystem/Zeeguu-Core/blob/master/zeeguu/model/language.py#L26

also, in the database, it makes little sense to have joined_language_ids. 
or
?